microsoft-edge/devtools-guide.md

```diff
@@ -94,4 +98,42 @@ These shortcuts control the main DevTools window and/or work across all tools.
 | Find in search box | `Ctrl`+`F` |  
 | Give focus to console at the bottom | `Alt`+`Shift`+`I` |  
 | Launch DevTools to Console | `Ctrl`+`Shift`+`J` |  
-| Refresh the page.  **NOTE**:  If you're debugging and paused at a breakpoint, this resumes execution first.  | `Ctrl`+`Shift`+`F5`, `Ctrl`+`R` |  
+| Refresh the page | `Ctrl`+`Shift`+`F5`, `Ctrl`+`R` |  
+
+> [!NOTE]
+> If you're debugging and paused at a breakpoint, the **Refresh the page** action resumes the runtime first.
+
+<!-- image links  -->  
+
+[ImageDevtoolsEdgehtml]: /microsoft-edge/devtools-guide/media/devtools.png "Microsoft Edge (EdgeHTML) DevTools"  
+[ImageDevtoolsGuideEdgehtmlChooselocal]: /microsoft-edge/devtools-guide/media/chooser_local.png "DevTools app Local panel"  
+[DevtoolsGuideEdgehtmlRemote]: ./devtools-guide/media/chooser_remote.png "DevTools app Remote panel"  
+
+<!-- links  -->  
+
+[DevtoolsGuideEdgehtmlConsole]: /microsoft-edge/devtools-guide/console "Console"  
+[DevtoolsGuideEdgehtmlDebugger]: /microsoft-edge/devtools-guide/debugger.md "Debugger"  
+[DevtoolsGuideEdgehtmlElements]: /microsoft-edge/devtools-guide/elements "Elements"  
+[DevtoolsGuideEdgehtmlEmulation]: /microsoft-edge/devtools-guide/emulation.md "Emulation"  
+[DevtoolsGuideEdgehtmlMemory]: /microsoft-edge/devtools-guide/memory.md "Memory"  
+[DevtoolsGuideEdgehtmlNetwork]: /microsoft-edge/devtools-guide/network.md "Network"  
+[DevtoolsGuideEdgehtmlPerformance]: /microsoft-edge/devtools-guide/performance.md "Performance"  
+[DevtoolsGuideEdgehtmlServiceworkers]: /microsoft-edge/devtools-guide/service-workers.md "Service Workers"  
+[DevtoolsGuideEdgehtmlStorage]: /microsoft-edge/devtools-guide/storage.md "Storage"  
+[DevtoolsGuideEdgehtmlWhatsnew]: /microsoft-edge/devtools-guide/whats-new "DevTools in the latest Windows 10 update (EdgeHTML 18)"  
+[DevtoolsProtocolEdgehtmlIndex]: /microsoft-edge/devtools-protocol/index.md "Microsoft Edge (EdgeHTML) DevTools Protocol"  
+[DevtoolsProtocolEdgehtmlClientsEdgePreview]: /microsoft-edge/devtools-protocol/0.1/clients.md#microsoft-edge-devtools-preview "Microsoft Edge DevTools Preview - DevTools Protocol Clients"  
+[HostingWebview]: /microsoft-edge/hosting/webview.md "WebView (EdgeHTML) for Windows 10 apps"  
+[PwasEdgehtmlIndex]: /microsoft-edge/progressive-web-apps-edgehtml/index.md "Progressive Web Apps (EdgeHTML) on Windows"  
+
+[MicrosoftDeveloperEdgeToolsRemote]: https://developer.microsoft.com/microsoft-edge/tools/remote/  
```

Comment:
Should we include a title?

Reply:
The link is bad, but this is old content and another link had not been provided.  
@zoherghadyali can you look into this?

Reply:
In context, I believe this link should point to: https://www.browserstack.com/test-on-microsoft-edge-browser

Reply:
Updated.
`[Test on Microsoft Edge \(EdgeHTML\) free from any browser][BrowserstackEdgehtml]`